dao-voting/tests/harness.rs

```diff
@@ -0,0 +1,974 @@
+#![allow(unused_variables)]
+#![allow(unused_imports)]
+
+use std::ops::Add;
```

Comment:
You can import directly from the library too huh. That's strange considering that this is not an input or output... How is this even working?

Reply:
Why is this needed?